src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -847,7 +847,7 @@ void PolicyHandler::OnPendingPermissionChange(
         app,
         mobile_apis::HMILevel::HMI_NONE,
         mobile_apis::AudioStreamingState::NOT_AUDIBLE,
-        true);
+        false);
```

Comment:
@LuxoftAKutsan 
I've discussed with Alex Galiuzov whether we have to send BC.ActivateApp notification and looks like we have since HMI needs to know that application has changed its level to NONE after it has been revoked. Otherwise HMI may continue to show it in, say, FULL level. I didn't find requirement on that so please create question to BA team to confirm that.